src/routes/(app)/+page.server.ts

```diff
@@ -1,9 +1,9 @@
 import type { RequestEvent } from '@sveltejs/kit';
 import type { PageServerLoad } from './$types';
-import type { stringsSchema } from '$lib/locale.store';
+import type { stringsSchema } from '../../lib/locale.store';
```

Comment:
Can we move this internal module to the group below? I try to keep the external modules separate from the internal modules.